tests/Integration/Behaviour/Features/Scenario/Shop/search_shops.feature

```diff
@@ -0,0 +1,33 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s shop --tags search-shops
+@reset-database-before-feature
+@clear-cache-before-feature
+@search-shops
+
+Feature: Search shops given a search term (BO)
+  As a BO user
+  I want to get a list of shops for a given search term
+
+  Background:
+    Given multiple shop context is loaded
+    Given shop "shop1" with name "test_shop" exists
+    Given I add a shop "shop2" with name "test_second_shop" and color "red" for the group "Default"
+    Given I add a shop "shop3" with name "test_third_shop" and color "blue" for the group "Default"
+
+  Scenario: I search for existing shops
+    When I search for shops with the term "test" I should get the following results:
```

Comment:
I feels weird to have empty colors in this test But the lines after you actually check that there are colors
In the internals of checking you just ignore when color is empty but it can be a valid test (if no color is set for example) So you should correctly set all the colors in all tests so that they remain consistent (especially if the value is empty)

Reply:
now I check colors all the time (even when its empty), but maybe (probably) it was not the case when you wrote this feedback